andela/travel_tool_back
# #160371487 Mark all notifications as read

#### What does this PR do?
* This PR updates the statuses of all notifications from `unread` to `read`.

#### Description of Task to be completed?
* write tests for the endpoint to update notifications statuses
* create endpoint and method to update notifications statuses
#### How should this be manually tested?
* Clone the repo -git clone https://github.com/andela/travel_tool_back.git
* Enter into the project directory - cd travel_tool_back
* Install project dependencies using `yarn install`
* Check `.env.example` and set up your `.env` appropriately
* Run database migrations using `yarn run db:migrate`
* Seed the database using `yarn run db:seed`
* Start the app using `make start`
* Navigate to [travela](http://travela-local.andela.com:3000/#) in your browser and login
* Login on your browser and grab your token from the application tab in Developers tab
* Open Postman
* Add your token in headers
* Send a PUT request to http://localhost:5000/api/v1/notifications with newStatus in req.body
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
* [#160371487](https://www.pivotaltracker.com/story/show/160371487)
#### Screenshots (if appropriate)
* None
#### Questions:
* None
